app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def increment_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#increment_comment_counter` method every time a new post has been saved to the database, then make the `#increment_comment_counter` method private